content/docs/hooks-state.md

```diff
@@ -8,7 +8,7 @@ prev: hooks-overview.html
 
 *Hooks* are a new addition in React 16.8. They let you use state and other React features without writing a class.
 
-The [previous page](/docs/hooks-intro.html) introduced Hooks with this example:
+The [introduction page](/docs/hooks-intro.html) used this example to introduce Hooks:
```

Comment:
How about you rephrase it?

```suggestion
The [introduction page](/docs/hooks-intro.html) used this example to get familiar with Hooks:
```

Reply:
I like that rephrasing!

Reply:
Great, you can apply it if you click on the "Commit suggestion" button.